metadata_xml/iso19115-cioos-template/main.j2

```diff
@@ -621,6 +626,59 @@
 			</mri:associatedResource>
       {% endfor %}
 
+      {% if record['identification']['language'] %}
+        {# defaultLocale: mandatory #}
+        <mri:defaultLocale>
+          <lan:PT_Locale>
+            {# language: mandatory #}
+            <lan:language>
+              <lan:LanguageCode codeList="http://standards.iso.org/iso/19115/resources/Codelist/lan/LanguageCode.xml" codeListValue="{{- resource_lang_code  -}}">
```

Comment:
if `resource_lang_code` is required, would be good to add to the if statement above. But maybe not required for validity. Also I thing `resource_lang` is an unused variable

Reply:
good point, It's a valid iso record without it. The CIOOS standard requires it however. I will include a check and not generate the xml snippet if false